torchvision/ops/deform_conv.py

```diff
@@ -50,7 +50,15 @@ def deform_conv2d(
         >>> weight = torch.rand(5, 3, kh, kw)
         >>> # offset and mask should have the same spatial size as the output
         >>> # of the convolution. In this case, for an input of 10, stride of 1
-        >>> # and kernel size of 3, without padding, the output size is 8
+        >>> # and kernel size of 3, without padding, the output size is 8.
+        >>> # For the simplest case, 1 channel 3x3 kernel, at each valid (consider padding) input feature position:
+        >>> # [[p0 p1 p2],
+        >>> #  [p3 p4 p5],
+        >>> #  [p6 p7 p8]]
+        >>> # The 18 corresponding offsets will be:
+        >>> # [p0_offset_h, p0_offset_w, p1_offset_h, p1_offset_w, p2_offset_h, ..., p8_offset_w]
+        >>> # ``_h`` means pixel offset in the height direction (i.e. p0 -> p6),
```

Comment:
"p0 - > p6" confuses me at the first glance. So maybe  "p0 -> p3 -> p6" is better.

Reply:
Good point! I'll change it in a bit.